### DISCUSSION TITLE
How to enable public pages, or what are they?

### DISCUSSION THREAD

**Discussion by pomtom44** [wavelog/wavelog#1234] [General]
I see reference to public pages a few times in the PR notes and issues / discussions, but I can't find anywhere how to turn them on or how they work.

is there some documentation im missing on this, because I feel like it would be a useful feature to have

**Comment by violarulan** [comment]
I think it is the public callbook. You can enable it in your station setup page.

**Comment by HB9HIL** [comment]
@pomtom44 

You can enable the public page in the station setup. It's part of a logbook


https://github.com/user-attachments/assets/329ccb8a-6fc4-48d0-ab7e-a79002ca4c20

The second part in the video, where I type in `staticmap/render/[slug]` will be available in the next release and generates a JPG Image from various options. Both, the `visitor/[]` and [staticmap](https://github.com/wavelog/wavelog/wiki/Static-Map-Images) are available without login and can safely exposed and shared.

**Reply by pomtom44** [reply]
Ah thank you, it was the station part I was missing.
I had found the flag in the global settings, and all the URL's in discussions, but nothing was working.
Tested and working fine now
